loading.bs

```diff
@@ -331,6 +390,433 @@ Note: Applying the signed exchange's response here has the effect of letting a
+        |outerLinkVariantKey|, |linkContext|) to |alternateLinks|.
+1. Let |allowedSxgLinks| be an empty [=list=].
+1. For each |innerLink| of |innerLinks|:
+    1. If |innerLink|'s [=Relation Type=] is not 'allowed-alt-sxg', then
+        continue.
+    1. Let |linkTarget| be |innerLink|'s [=Link Target=].
+    1. Let |innerLinkVariants| be the second item of the first tuple of
+        |innerLink|'s [=Target Attributes list|Target Attributes=] whose first
+        item matches the string `"variants"` or the empty string ("") if it is
+        not present.
+    1. Let |innerLinkVariantKey| be the second item of the first tuple of
+        |innerLink|'s [=Target Attributes list|Target Attributes=] whose first
+        item matches the string `"variant-key"` or the empty string ("") if it
+        is not present.
+    1. Let |headerIntegrity| be the second item of the first tuple of
+        |innerLink|'s [=Target Attributes list|Target Attributes=] whose first
+        item matches the string `"header-integrity"` or continue if it is not
+        present.
+    1. [=list/Append=] (|linkTarget|, |innerLinkVariants|,
+        |innerLinkVariantKey|, |headerIntegrity|) to |allowedSxgLinks|.
+1. For each |innerLink| of |innerLinks|:
+    1. If |innerLink|'s [=Relation Type=] is not 'preload', then continue.
+    1. Let |linkTarget| be |innerLink|'s [=Link Target=].
+    1. Let |asAttribute| be the second item of the first tuple of |innerLink|'s
+        [=Target Attributes list|Target Attributes=] whose first item matches
+        the string `"as"` or the empty string ("") if it is not present.
+    1. If |asAttribute| is `"image"`, then:
+        1. Let |imagesrcset| be the second item of the first tuple of
+            |innerLink|'s [=Target Attributes list|Target Attributes=] whose
+            first item matches the string `"imagesrcset"` or the empty string
+            ("") if it is not present.
+        1. Let |imagesizes| be the second item of the first tuple of
+            |innerLink|'s [=Target Attributes list|Target Attributes=] whose
+            first item matches the string `"imagesizes"` or the empty string
+            ("") if it is not present.
+        1. Create a <{link}> element |linkElement| which <{link/href}> attribute
+            is |linkTarget|, and <{link/imagesrcset}> attribute is
+            |imagesrcset|, and <{link/imagesizes}> is |imagesizes|.
+        1. Let |selected source| and <var ignore=''>selected pixel density</var>
+            be the URL and pixel density that results from [=selecting an image
+            source=] given |linkElement|, respectively.
+        1. If |selected source| is not null, then set |linkTarget| to the result
+            of [=URL parser|parsing=] |selected source|, with a base URL of
+            |response|'s [=response/URL=].
+    1. For each |allowedSxgLink| of |allowedSxgLinks|:
+        1. If the first element isn't the same as |linkTarget|, then continue.
```

Comment:
```suggestion
        1. If |allowedSxgLink|'s linkTarget isn't the same as |linkTarget|, then continue.
```

It might be good to explicitly define the Allowed SXG Link struct so that you could link to its fields in the above line.